secp256k1-sys/depend/scratch_impl.h.patch

```diff
@@ -1,26 +0,0 @@
-13,37d12
```

Comment:
Instead of making `secp256k1-zkp` a submodule, it would be easier to track changes against upstream with those files inline.

What are your thoughts on including `secp256k1-zkp` in a future version of this PR?

Reply:
Yeah I'm wondering that too. I'd be closer to upstream too.